imports/plugins/included/product-variant/components/productGrid.js

```diff
@@ -218,9 +215,78 @@ class ProductGrid extends Component {
           </Menu>
         </Toolbar>
       );
-    }
+    // eslint-disable-next-line no-else-return
```

Comment:
This doesn't need to be inside of an `else`, you can just `return` immediately after the `if`, since there isn't any other option.